vignettes/epinow.Rmd

```diff
@@ -51,15 +51,12 @@ res <- epinow(reported_cases,
 )
 #> Logging threshold set at INFO for the name logger
 #> Writing EpiNow2 logs to the console and:
-#> '/tmp/RtmpM4AQFy/regional-epinow/2020-04-21.log'.
+#> '/tmp/Rtmp5jGZoX/regional-epinow/2020-04-21.log'.
 #> Logging threshold set at INFO for the name logger
 #> Writing EpiNow2.epinow logs to the console and:
-#> '/tmp/RtmpM4AQFy/epinow/2020-04-21.log'.
-#> WARN [2024-12-06 15:12:56] epinow: There were 2 divergent transitions after warmup. See
-#> https://mc-stan.org/misc/warnings.html#divergent-transitions-after-warmup
-#> to find out why this is a problem and how to eliminate them. - 
-#> WARN [2024-12-06 15:12:56] epinow: Examine the pairs() plot to diagnose sampling problems
-#>  -
+#> '/tmp/Rtmp5jGZoX/epinow/2020-04-21.log'.
+#> WARN [2025-01-01 00:30:56] epinow: The `filter_leading_zeros` argument of `estimate_infections()` is deprecated as of EpiNow2 1.7.0. - 
+#> WARN [2025-01-01 00:30:56] epinow: The `zero_threshold` argument of `estimate_infections()` is deprecated as of EpiNow2 1.7.0. -
```

Comment:
We need to remove the use of these arguments to remove the deprecation warnings.